test/suite/wttest.py

```diff
@@ -766,6 +773,8 @@ def runsuite(suite, parallel):
         WiredTigerTestCase._concurrent = True
         suite_to_run = ConcurrentTestSuite(suite, fork_for_tests(parallel))
     try:
+        if WiredTigerTestCase._randomseed:
+            WiredTigerTestCase.prout("Starting test suite with seedw=" + str(WiredTigerTestCase._seeds[0]) + " and seedz=" + str(WiredTigerTestCase._seeds[1]))
```

Comment:
Assuming you combine the seeds suggested above, I would say: `Rerun this test or any part of it with "--seed wwwww.zzzzzz" to get the same randomness` .  That gives the user clear instructions on what to do with the information.

Reply:
done!